### PR TITLE
Imported `components/buttonBtn.less`

### LINKED ISSUE #4933: Log In + Sign Up link styling broken on Book pages
@Yashs911 reported 14 days ago:
CSS for Log In and Sign Up seems to be missing from the Books Page (http://staging.openlibrary.org/works/OL6384123W/Self-knowledge and now on production https://openlibrary.org/works/OL6384123W/Self-knowledge)

### Evidence / Screenshot (if possible)
openlibrary org_works_OL6384123W_Self-knowledge

### Relevant url?

https://openlibrary.org/works/OL6384123W/Self-knowledge

### Steps to Reproduce

1. Go to ... https://openlibrary.org/works/OL6384123W/Self-knowledge) incognito

### Details

- **Logged in (Y/N)?**  N
- **Browser type/version?** chrome 
- **Operating system?**  linux
- **Environment (prod/dev/local)?** prod + staging

### Stakeholders

@cdrini @jamesachamp @jdlrobson j

### PR DESCRIPTION
**Pull request by soham4abc**
Closes #4933 

### Stakeholders
@jdlrobson

### CODE REVIEW

File: static/css/page-book.less [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 @import (less) "components/metadata-form.less";
 @import (less) "components/observationStats.less";
+@import "components/buttonBtn.less";
```
Comment (Yashs911) [inline static/css/page-book.less:31]:
```suggestion
@import (less) "components/buttonBtn.less";
```

Comment (Yashs911) [reply static/css/page-book.less:31]: According to the update on https://github.com/internetarchive/openlibrary/issues/4933#issuecomment-809646182 the component should be imported by `header.less` but importing `components/buttonBtn.ess` directly solved the issue 🎉

Comment (soham4abc) [reply static/css/page-book.less]: All okay now?

**Comment by jdlrobson** [comment]
Python issue unrelated.

**Pull request merged**